pkgs/by-name/ha/halo/package.nix

```diff
@@ -0,0 +1,61 @@
+{ lib
+, stdenv
+, fetchurl
+, makeWrapper
+, temurin-jre-bin
+, rsync
+, ...
+}:
+stdenv.mkDerivation rec {
+  pname = "halo";
+  version = "2.15.1";
+  src = fetchurl {
+    url = "https://github.com/halo-dev/halo/releases/download/v${version}/${pname}-${version}.jar";
+    hash = "sha256-6qgbUXd0E38Rhv0Jf1qP54yJhQiB1a/Jr1FYw3bC4qI=";
+  };
+
+  nativeBuildInputs = [
+    makeWrapper
+    rsync
+  ];
+
+  buildInputs = [
+    temurin-jre-bin
+  ];
+
```

Comment:
```suggestion
```

Reply:
This should be put in nativeBuildInputs.

Reply:
ok i got